pkg/helm/helmcli.go

```diff
@@ -76,9 +76,7 @@ func SetDefaultLoadChartFunction() {
 
 // GetRelease will run 'helm get all' command and return the output from the command.
 func GetRelease(kubeInfo *client.KubeInfo, releaseName string, namespace string) (*release.Release, error) {
-	settings := cli.New()
-	settings.SetNamespace(namespace)
```

Comment:
Is this the only setting we ever set?

Reply:
In most cases, this is the only setting that we set. In one or two cases, we set other options when dealing with providers. In most cases we would set the namespace setting, but not pass it in.

Reply:
but you are not passing that settings param anymore, doesn't this affect the scenario where this is used for providers?